components/schedule-editor/src/styles/schedule-editor.scss

```diff
@@ -107,6 +144,113 @@ section {
       }
     }
 
+    .edit-custom-field {
+      margin: 0 1rem;
+
+      div {
+        margin: 0.5rem 2em;
+        text-align: left;
+
+        &.header {
+          display: flex;
+          justify-content: space-between;
+
+          .delete {
+            background-color: white;
+            color: var(--red);
+            flex: 0.25;
+            font-size: 16px;
+            font-weight: normal;
+            margin-bottom: 0.8rem;
+
+            span {
+              margin-left: 0.5rem;
+            }
```

Comment:
I'm seeing a lot of redundant CSS with `button.edit` [above](https://github.com/nylas/components/pull/375/files?diff=split&w=1#diff-99e184586bca18273db219244222807128ffa8fd187dc5ae6a3691ce7f496726R123-R134) and `button.delete`. I think there could be shared properties that are not nested as deep. The span with margin-left also appears twice. That can probably just be moved up to be a child of the button css